### DISCUSSION TITLE
is there a way to format datetimes?

### DISCUSSION THREAD

**Discussion by trondhindenes** [piccolo-orm/piccolo_admin#371] [General]
Hi, 
Realizing I've asked about this before, but I can't see a way to affect the formatting og datetime - I'd really like to be able to show a 24-hour clock instead of the default 12-hours. Is that possible?

**Comment by dantownsend** [comment]
We use the default [`time` HTML widget](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time) - I think it's browser dependent whether it shows a 24 hour clock or not.

You can set the [time resolution](https://piccolo-admin.readthedocs.io/en/latest/table_config/index.html?highlight=resolution#source) using `TableConfig`, which can be used to disable seconds / milliseconds etc if that's any help.

When values are listed in the row page, I don't think we do any formatting of dates / times at the moment - just the raw value is shown.

**Comment by trondhindenes** [comment]
thanks. Its the good old language vs locale issue I guess. As many others in non-english-native countries I use US english as the _language_ but not the _locale_. Lots of systems fail to appreciate the subtle difference, it seems `time` is one of them.

Edit: checking now, and the browser actually returns us-en locale. I don't think Chrome distinguishes between language and locale.
